lib/dish.rb

```diff
@@ -0,0 +1,10 @@
+class Dish 
+  attr_reader :name,
+              :category 
+
+  def initialize(name = "Couscous Salad", category)
+    @name = name
+    @category = :appetizer 
```

Comment:
This will make every dish an appetizer, instead use this: `@category = category`